src/modules/Packages.rb

```diff
@@ -2702,6 +2700,16 @@ def format_missing_resolvables(type, list)
         _("These items (%{type}) need to be selected to install: %{list}") % {type: type, list: list}
       end
     end
+
+    def fs_mountpoint(filesystem)
+      filesystem.mountpoints[0] || ""
+    end
+
+    def fs_name(filesystem)
```

Comment:
is it fs name or device name? as it looks more for me like name of block device, not?